Common/include/CConfig.hpp

```diff
@@ -800,7 +800,7 @@ class CConfig {
   Beta_Factor,          /*!< \brief Value of the epsilon^2 multiplier for Beta for the incompressible preconditioner. */
   Gas_Constant,         /*!< \brief Specific gas constant. */
   Gas_ConstantND,       /*!< \brief Non-dimensional specific gas constant. */
-  Molecular_Weight,     /*!< \brief Molecular weight of an incompressible ideal gas (g/mol). */
+  *Molecular_Weight,    /*!< \brief Molecular weight of an incompressible ideal gas (g/mol). */
   Specific_Heat_Cp,           /*!< \brief Specific heat at constant pressure. */
```

Comment:
Hi Tobias, Thank you for the feedback, I have moved nMolecular_Weight to line 831, because When I add it in line 806, SU2 does not compile because the variables above line 806 are doubles so adding it at line 806 causes that the variables below that line become undefined. Please let me know what you think.

Reply:
Oh right. Yeah from my point of view it is ok to break the su2double-train and insert variables where they fit. I adapted my code suggestion to be hopefully correct now :)